docs/releases/1.10.txt

```diff
@@ -55,6 +55,31 @@ Minor features
 :mod:`django.contrib.auth`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Support for the Argon2_ password hash has been added.  Argon2 is the winner
+  of the recent `Password Hashing Competition`_, which has been organized
+  by the community in similar style as the NIST competitions that selected
+  the hash SHA-3 and cipher AES.
+
+  To use Argon2_ put ``django.contrib.auth.hashers.Argon2Hasher`` at the start
+  of :setting:`PASSWORD_HASHERS`.  That is, in your settings file, you'd put::
+
+      PASSWORD_HASHERS = [
+          'django.contrib.auth.hashers.Argon2PasswordHasher',
+          'django.contrib.auth.hashers.PBKDF2PasswordHasher',
+          'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+          'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
+          'django.contrib.auth.hashers.BCryptPasswordHasher',
+          'django.contrib.auth.hashers.SHA1PasswordHasher',
+          'django.contrib.auth.hashers.MD5PasswordHasher',
+          'django.contrib.auth.hashers.UnsaltedSHA1PasswordHasher',
+          'django.contrib.auth.hashers.UnsaltedMD5PasswordHasher',
+          'django.contrib.auth.hashers.CryptPasswordHasher',
+      ]
+
+  You need to keep the other entries in this list, or else Django won't
```

Comment:
I would link to "argon2_usage" instead of repeating the example usage parts of it in the release notes.